core/backend/src/test/TestUtils.ts

```diff
@@ -105,4 +105,3 @@ before(async () => {
 after(async () => {
   await TestUtils.shutdownBackend();
 });
-
```

Comment:
inadvertent change?

Reply:
It was unintentional. Thanks for pointing it out